Running_Linux_OSX.txt

```diff
@@ -59,8 +60,9 @@ Autopsy depends on a specific version of The Sleuth Kit.  You need the Java libr
 
 * Running Autopsy *
 
-- In a terminal, change to the ‘bin’ directory in the Autopsy folder.
-- Run Autopsy
+- The Autopsy executable is in the ‘bin’ directory in the Autopsy folder.
```

Comment:
I don't have much of an issue with this change except that it now doesn't align with unix_setup.sh which when you run it states "Autopsy is now configured. You can execute bin/autopsy to start it"